firmware/Core/Inc/flash_bank/flash_bank.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __INCLUDE_GUARD__FLASH_BANK_H__
+#define __INCLUDE_GUARD__FLASH_BANK_H__
+#include <stdint.h>
+
+/// @brief  Flash Partitions
+/// @note look in the STM32L4R5ZITx_FLASH.ld file
+/// to see the address of each partition, update this as needed
+typedef enum
+{
+    Flash_Partitions_RAM = 0x20000000,
+    Flash_Partitions_RAM_2 = 0x10000000,
+    Flash_Partitions_RAM_3 = 0x20040000,
+    Flash_Partitions_FLASH_BANK1 = 0x8000000, // default FLASH address
+    Flash_Partitions_FLASH_BANK2 = 0x8100000
+} Flash_Partition_Adressess;
```

Comment:
renamed as suggested, using memory region instead of partition. You are right in pointing out that generally, these areas are referred to memory regions, so it makes sense to keep it consistent